lib/coverband/tasks.rb

```diff
@@ -20,20 +20,23 @@
             require baseline_file
           end
         end
-        if defined? Rails
-          Dir.glob("#{Rails.root}/app/**/*.rb").sort.each { |file| 
+        if defined? Rails && !Rails.config.eager_load
+          Dir.glob("#{Rails.root}/app/**/*.rb").sort.each do |file|
               begin
                 require_dependency file
               rescue Exception
                 #ignore
-              end }
+              end
+          end
+
           if File.exists?("#{Rails.root}/lib")
```

Comment:
nah - both of these things are skipped if eager load is true or if we're not in Rails.  some people (yours truly) stupidly keep destructive files not loaded in the normal course of things in the lib directory.  ;)